app/code/Magento/Newsletter/view/adminhtml/layout/newsletter_subscriber_block.xml

```diff
@@ -91,7 +91,6 @@
                         <arguments>
                             <argument name="header" xsi:type="string" translate="true">Customer First Name</argument>
                             <argument name="index" xsi:type="string">firstname</argument>
-                            <argument name="default" xsi:type="string">----</argument>
```

Comment:
@edenduong Do you think removing `----` and keeping this argument tag as it is without value would also solve the issue?

Reply:
Hi @kalpmehta : Yes, It can solve this issue. But I think it would be better if we remove it. 
Thank you .